### PR TITLE
Seismicity time

### PR DESCRIPTION
**Pull request by LucaDeSiena**
Adding tutorial to create videos, based on the Campi Flegrei dataset.

+ Tutorial for temporal distribution of seismicity.

- Movie_Paraview creates the movie but gives a strange error at the end.

? Could we change Movie_Paraview to create automatically a new folder to store frames, like in the tutorial? Otherwise, the folder becomes full of .vts

### CODE REVIEW

**Comment by deconvolution** [comment]
Hi Luca,

I have made a tutorial that creates output in a new folder and does not make any errors. But I edited based on your previous tutorial. With this one, you can have an output folder like this. The .vtu files are not messing up the folder. Documentation is also added. You can check if everything is fine. I will push this after you take a look.
https://github.com/deconvolution/GeophysicalModelGenerator.jl/tree/seismicitytime

[cid:dd876650-0598-4df6-8d02-f2a71576706a]

I found something inconsistent with the input file that causes some problems in identifying the date. For instance, the time value in line 8 is not consistent with that line 7 (length of integers). I wrote a script to correct this. Now I have 2 corrected outputs. Can you upload those 2 files to the link to replace the original file?
[cid:65e11781-abbc-45b5-a320-5f0e15035b76]


Best,

Yi
________________________________

**Comment by LucaDeSiena** [comment]
Thanks Yi,

in this pull request you will find a more general tutorial for including movies, [here](https://github.com/LucaDeSiena/GeophysicalModelGenerator.jl/blob/SeismicityTime/docs/src/man/tutorial_time_Seismicity.md).

I created it as a stand-alone example of how to do movies, which you will widen with the wave propagation. It includes a solution to deal with the 1980s data. We can leave the 2000s data as an example in Flegrei, but it is necessary to show the seismicity with the ambient noise map and the geomorphology.

Let's wait for Boris to check and approve this one, then we can work on your version to make it consistent.

@boriskaus - it is maybe time to split the volcano examples from those more general or for the Alps in the manual?

File: docs/src/man/tutorial_time_Seismicity.md
Comment (boriskaus) [inline docs/src/man/tutorial_time_Seismicity.md:74]: If I understand it correctly, the data you actually read into `GMG` is in UTM coordinates. If that is the case, please load it into a `UTMData` structure instead You can always convert this to the correct `CartData` structure using a `ProjectionPoint` and `Convert2CartData`.

I recall that your argument to not do this is that the `ShapeFiles` in paraview are only shown in `UTM` coordinates. This can, in fact, be transferred to the correct distances, by shifting that data in Paraview and dividing it by 1000 (using the `Transform` filter.

Things have to be in the correct `CartData` form to generate consistent LaMEM model input.

File: tutorial/Tutorial_SeismicityTime.jl
Comment (boriskaus) [inline tutorial/Tutorial_SeismicityTime.jl:53]: Same comment here; this is a UTM dataset and not a CartData set

**Review by boriskaus — CHANGES_REQUESTED** [review]

**Comment by boriskaus** [comment]
@LucaDeSiena As to your comment on splitting the volcano tutorials from the Alpine ones: I think that tutorials should explain how to load a new type of data, or how to use several functions (not yet covered in others). To which geographic area this is applied is not important. Once users have worked through all tutorials they should have the knowledge to apply all of this to their region of interest

**Comment by boriskaus** [comment]
I'm happy to approve it now, but I am a bit worried that the test for the `ProjectCartData` is not working in combination with a `UTMData` dataset. I will have a look at it after this is merged.

**Review by boriskaus — APPROVED** [review]

**Pull request merged**
